src/utils/coingecko.ts

```diff
@@ -29,3 +29,22 @@ export async function getTokensUSDPrice(by: GetPriceBy, addressesOrIds: string[]
     return prices
   }
 }
+
+export async function getInitialTokenUSDPrice(symbol: string, date: string) {
```

Comment:
Suggestion for a better name

```suggestion
export async function getTokenUSDPriceAtDate(symbol: string, date: string) {
```